h11/_headers.py

```diff
@@ -74,6 +74,11 @@ def normalize_and_validate(headers):
         # downstream." -- https://tools.ietf.org/html/rfc7230#section-3.2.4
         if name.strip() != name:
             raise LocalProtocolError("Illegal header name {}".format(name))
+        # "A sender MUST NOT generate a message that includes line folding"
+        # -- https://tools.ietf.org/html/rfc7230#section-3.2.4
```

Comment:
Ok cool, that was my understanding from our first pass in Requests. Just wanted to clarify I wasn't missing a more direct mandate.